rebasebot/github.py

```diff
@@ -1,3 +1,16 @@
+#    Copyright 2022 Red Hat, Inc.
```

Comment:
Dunno if it's important or not, but this was added in 2023

Reply:
i'll update the tests files and this one, thanks!